juju/home.go

```diff
@@ -23,5 +26,16 @@ func InitJujuXDGDataHome() error {
 	if err := ssh.LoadClientKeys(osenv.JujuXDGDataHomePath("ssh")); err != nil {
 		return errors.Annotate(err, "cannot load ssh client keys")
 	}
+
+	base := osenv.JujuXDGDataHomePath("x509")
+	keyPath := filepath.Join(base, "winrmclientkey.pem")
+	certPath := filepath.Join(base, "winrmclientcert.crt")
+	cert := winrm.NewX509()
+	if err := cert.LoadClientCert(certPath, keyPath); err != nil {
+		return errors.Annotatef(err, "connot load/create x509 client certs for winrm connection")
+	}
+
+	_ = cert.LoadCACert(filepath.Join(base, "winrmcacert.crt"))
```

Comment:
I'm skipping this error because this ```InitJujuXDGDataHome``` runs everytime I launch a juju command. So not all commands deal with provisioning or needs that CA cert to work. This is also very "windows specific" and only the juju add-machine benefits from this. So we don't need to annoy the user that he doesn't have a CACert alongside with the client cert and key.
This check is already done in the winrmprovisioner code when the provisioning process starts.